ChallengesWithTestsMarkVII/ChallengesSet07.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace ChallengesWithTestsMarkVII
+{
+    public class ChallengesSet07
+    {
+        public int CountOfBusinessesWithNegativeNetProfit(List<Business> businesses)
+        {
+            if (businesses == null)
+            {
+                return 0;
+            }
+            int count = 0;
+            foreach (Business biz in businesses)
+                if (biz.TotalExpenses < biz.TotalRevenue)
+                {
+                    count += 0;
```

Comment:
This line isn't needed. Consider changing the if statement around so that only the `count += 1;` line is involved, `if (biz.TotalExpenses > biz.TotalRevenue) { count += 1; }`